src/test/java/com/epam/tc/hw2/BrowserTestBase.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.tc.hw2;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.ITestResult;
+import org.testng.Reporter;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+
+public class BrowserTestBase {
+
+    protected WebDriver driver;
+
+    // 1. Open test site by URL
+    @BeforeClass
+    public void setup() {
+        ITestResult itr = Reporter.getCurrentTestResult();
+        System.err.println("ClassName: " + itr.getInstance().getClass().getName());
+        driver = WebDriverManager.chromedriver().create();
```

Comment:
Could be simplified...
        WebDriverManager.chromedriver().setup();
        driver = new ChromeDriver();